website/docs/d/cloud_project_user_s3_credential.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "ovh"
+page_title: "OVH: cloud_project_user_s3_credential"
+sidebar_current: "docs-ovh-datasource-cloud-project-user-s3-credential"
+description: |-
+  Get the Secret Access Key of an Access Key ID of a public cloud project's user.
+---
+
+# ovh_cloud_project_user_s3_credential (Data Source)
+
+Use this data source to retrieve the Secret Access Key of an Access Key ID associated with a public cloud project's user.
+
+## Example Usage
+
+```hcl
+data "ovh_cloud_project_user_s3_credentials" "s3_credentials" {
```

Comment:
Useless if we set manually the user_id in the `ovh_cloud_project_user_s3_credential` datasource.